ocpmodels/modules/scheduler.py

```diff
@@ -16,11 +16,14 @@ def __init__(self, optimizer, config):
             scheduler_lambda_fn = lambda x: warmup_lr_lambda(x, self.config)
             self.config["lr_lambda"] = scheduler_lambda_fn
 
-        self.scheduler = getattr(lr_scheduler, self.scheduler_type)
-        scheduler_args = self.filter_kwargs(config)
-        self.scheduler = self.scheduler(optimizer, **scheduler_args)
+        if self.scheduler_type != "Null":
+            self.scheduler = getattr(lr_scheduler, self.scheduler_type)
+            scheduler_args = self.filter_kwargs(config)
+            self.scheduler = self.scheduler(optimizer, **scheduler_args)
```

Comment:
Could we add a comment at the beginning of this class somewhere that if no scheduler is specified, the default scheduler is warmup_lr_lambda for backward-compatibility reasons and not no scheduler. And hence we need a separate "Null" scheduler if someone really doesn't want a scheduler.

Reply:
Yup, I will add that